modules/migration-combining-must-gather.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="migration-combining-must-gather_{context}"]
+= Combining options when using the must-gather tool
+
+Currently, it is not possible to combine many gather scripts, for example specifying a timeout threshold while permitting insecure TLS connections. Nonetheless, it is feasible to circumvent this restriction in certain instances by establishing internal variables on the must-gather command line, such as:
```

Comment:
Lastly, the [IBM style guide asks us](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording) to avoid introducing lists with phrases instead of complete sentences: "To avoid possible translation problems, use complete sentences to introduce lists and not phrases. Machine translators often translate at the sentence-level and return the best results when they have a complete sentence to parse."

```suggestion
Currently, it is not possible to combine many gather scripts, for example specifying a timeout threshold while permitting insecure TLS connections. Nonetheless, it is feasible to circumvent this restriction in certain instances by establishing internal variables on the must-gather command line, such as the following example:
```

Reply:
It looks like this feedback was resolved without being incorporated.